server/handler/static_files.go

```diff
@@ -10,18 +10,20 @@ import (
 
 // Static contains handlers to serve static using go-bindata
 type Static struct {
-	dir       string
-	serverURL string
+	dir          string
+	serverURL    string
+	GaTrackingID string
 }
 
 // NewStatic creates new Static
-func NewStatic(dir, serverURL string) *Static {
-	return &Static{dir, serverURL}
+func NewStatic(dir, serverURL, GaTrackingID string) *Static {
+	return &Static{dir, serverURL, GaTrackingID}
 }
 
 type options struct {
-	ServerURL   string      `json:"SERVER_URL"`
-	InitalState interface{} `json:"initialState"`
+	ServerURL    string      `json:"SERVER_URL"`
+	GaTrackingID string      `json:"GA_TRACKING_ID"`
```

Comment:
This value was not passed to the [marshalled options](https://github.com/src-d/code-annotation/pull/162/files#diff-33aa342cfbbafa0c5a364a50258cd956R50), so it does not get the frontend

Reply:
fixed. Thanks a lot!